tests/test_player_stats.py

```diff
@@ -91,3 +91,28 @@ def test_detect_complex_inf():
     })
 
     assert p.detect_team() == PlayerTeamAssociation(side=Team.ALLIES, confidence=PlayerTeamConfidence.STRONG, ratio=100)
+
+
+def test_detect_ignores_some_other_side():
+    p = PlayerStats(weapons={
+        'M1 GARAND': 45,
+        'M1A1 THOMPSON': 4,
+        'MK2 GRENADE': 3,
+        'BAZOOKA': 2,
+    }, death_by_weapons={
+        'GEWEHR 43': 28,
+        'MP40': 7,
+        'KARABINER 98K': 7,
+        'FG42 x4': 6,
+        '150MM HOWITZER [sFH 18]': 5,
+        'STG44': 3,
+        'COAXIAL MG34 [Sd.Kfz.181 Tiger 1]': 2,
+        '88 KWK 36 L/56 [Sd.Kfz.181 Tiger 1]': 2,
+        'MG42': 1,
+        'SATCHEL': 1,
+        'BOMBING RUN': 1,
+        # this is a teamkill or a wrongly attributed kill and should not result in a MIXED team confidence
```

Comment:
it's not a teamkill nor wrongly attributed in my opinion. the player was probably killed by an explosion of the Sherman which must have been caused by any kind of explosive used by the opposite team eg. artillery, tank, satchel, at gun, ...

Reply:
let me add that as additional info to the comment 👍